### PR TITLE
PLNSRVCE-37 - Package and run Tekton Pipeline with KCP running in k8s

### PR DESCRIPTION
**Pull request by bnallapeta**
- Modified the run.sh to install pipelines based on user args.
- Installs pipelines and runs example TaskRuns and PipelineRuns.

### CODE REVIEW

File: ckcp/run.sh [hunk 1]
```diff
@@ -58,3 +58,91 @@ KUBECONFIG=kubeconfig/admin.kubeconfig kubectl api-resources
...
+
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl apply $(ls pipeline/config/config-* | awk ' { print " -f " $1 } ')
+
+      # Test 3 - create taskrun and pipelinerun
+
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl create serviceaccount default
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl create -f pipeline/examples/v1beta1/taskruns/custom-env.yaml
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl create -f pipeline/examples/v1beta1/pipelineruns/using_context_variables.yaml
+
+      KUBECONFIG=../kubeconfig/admin.kubeconfig METRICS_DOMAIN=knative.dev/some-repository SYSTEM_NAMESPACE=tekton-pipelines KO_DATA_PATH=./pipeline/pkg/pod/testdata ./pipeline/bin/controller \
```
Comment (vdemeester) [inline ckcp/run.sh:117]: - `METRICS_DOMAIN` could point to something like `openshift.io/pipeline-service` or something (even though not really useful today).
- Flags might evolve over time upstream, right now it got "extracted" manually right ? I wonder if we could have a small "script" that extract the command/args from the release.yaml.

File: ckcp/run.sh [hunk 2]
```diff
@@ -58,3 +58,91 @@ KUBECONFIG=kubeconfig/admin.kubeconfig kubectl api-resources
...
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-resource.yaml
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-condition.yaml
+
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl apply $(ls pipeline/config/config-* | awk ' { print " -f " $1 } ')
+
+      # Test 3 - create taskrun and pipelinerun
+
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl create serviceaccount default
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl create -f pipeline/examples/v1beta1/taskruns/custom-env.yaml
+      KUBECONFIG=../kubeconfig/admin.kubeconfig kubectl create -f pipeline/examples/v1beta1/pipelineruns/using_context_variables.yaml
```
Comment (vdemeester) [inline ckcp/run.sh:115]: Any reason to create the taskrun/pipelinerun before running the controller ?

**Review by vdemeester — COMMENTED** [review]
SGTM, just few comments 👼🏼

File: ckcp/run.sh (hunk 1 above)
Comment (guillaumerose) [inline ckcp/run.sh:117]: We need to run this command in a container so that when the script ends, it still works.

Comment (bnallapeta) [reply ckcp/run.sh:117]: Thanks for this idea. I like extracting info from latest release.yaml and using it here. I will work on this and raise another PR. For now, have raised this PR to accommodate running pipelines.

https://github.com/openshift-pipelines/pipelines-service/pull/18

Comment (bnallapeta) [reply ckcp/run.sh:117]: https://github.com/openshift-pipelines/pipelines-service/pull/18

**Comment by bnallapeta** [comment]
Closing this PR as I opened a new one
https://github.com/openshift-pipelines/pipelines-service/pull/18

**Pull request closed without merging**